nh_ews/ews.py

```diff
@@ -781,18 +810,27 @@ def get_last_case(self, cr, uid, patient_id, context=None):
         :returns: ``False`` or the acuity case
         :rtype: int
         """
+        last_obs_activity = self.get_last_obs_activity(cr, uid, patient_id)
+        case = int(self._POLICY['case'][bisect.bisect_left(
+            self._POLICY['ranges'], last_obs_activity.data_ref.score)])
+        case = 2 if last_obs_activity.data_ref.three_in_one and case < 3 else case
+        return case
+
+
+    def get_last_obs_activity(self, cr, uid, patient_id, context=None):
         domain = [['patient_id', '=', patient_id],
                   ['data_model', '=', 'nh.clinical.patient.observation.ews'],
                   ['state', '=', 'completed'],
                   ['parent_id.state', '=', 'started']]
+
         activity_pool = self.pool['nh.activity']
         ews_ids = activity_pool.search(
             cr, uid, domain, order='date_terminated desc, sequence desc',
             context=context)
-        if not ews_ids:
-            return False
-        activity = activity_pool.browse(cr, uid, ews_ids[0], context=context)
-        case = int(self._POLICY['case'][bisect.bisect_left(
-            self._POLICY['ranges'], activity.data_ref.score)])
-        case = 2 if activity.data_ref.three_in_one and case < 3 else case
-        return case
+        if ews_ids:
```

Comment:
Need to return falsde if no ews_ids in get last obs and also return false in get last case